pkg/apis/v1alpha1/format.go

```diff
@@ -1,5 +1,5 @@
 package v1alpha1
 
 // Format determines the output format (json or yaml).
-// +kubebuilder:validation:Enum:=json;yaml;
+// +kubebuilder:validation:Enum=json;yaml;name;go-template;go-template-file;template;templatefile;jsonpath;jsonpath-as-json;jsonpath-file
```

Comment:
We don't support all formats

Reply:
:(

Reply:
We can create a tracking issue but we don't really have use case for more complex formats today so 🤷